R/Figures.R

```diff
@@ -1,15 +1,17 @@
 #' Create Boxplot
 #' 
 #' @param filtered.data TADA data frame containing the data downloaded from the WQP, where
-#' each row represents a unique data record. Data frame must include the columns
-#' 'TADA.ComparableDataIdentifier", 'TADA.ResultMeasureValue', and
-#' 'TADA.ResultMeasure.MeasureUnitCode' to run this function. These columns can 
+#' each row represents a unique data record. Data frame must include the columns 'TADA.CharacteristicName' OR
```

Comment:
Let's discuss this at our next meeting (i.e. using only TADA.ComparableDataIdentifier for figure and stat generation, or allowing for TADA.CharacteristicName to be used on its own. There are important pros and cons to consider.